include/libsemigroups/present.tpp

```diff
@@ -453,6 +453,9 @@ namespace libsemigroups {
     void replace_subword(Presentation<W>& p,
                          W const&         existing,
                          W const&         replacement) {
+      if (existing.cbegin() == existing.cend()) {
```

Comment:
I originally tried `if (existing == word_type({}))`, but there was a type mismatch I wasn't confident about fixing so I did it this way. I'm completely open to suggestion here, since I don't think this is as elegant as could be!

Reply:
Should probably be `if (existing.empty()) {`